torch/distributed/rpc/api.py

```diff
@@ -259,35 +264,57 @@ def remote(to, func, args=None, kwargs=None):
     kwargs = kwargs if kwargs else {}
 
     info = _to_worker_info(to)
+    current_worker = get_worker_info()
+    # If profiling is enabled, kick off the timer and retrieve back a
+    # RecordFunction instance.
+    rf = None
+    if torch.autograd._profiler_enabled():
+        rf = _start_record_function(
+            RPCExecMode.REMOTE,
+            str(qualified_name if qualified_name is not None else func.__name__),
```

Comment:
nit: does func.__name__ give the fully qualified name of the function with the package? (ex: foo.bar.baz.my_func) If not can we make this fully qualified?

Reply:
it does not, though it looks like there is `func.__qualname__` that we can use as of Python 3.3 that should give us this: https://docs.python.org/3/glossary.html#term-qualified-name